src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx

```diff
@@ -24,9 +26,9 @@ export type BannerContent =
 export type DashboardTopBannerProps = {
   content: BannerContent;
   bannerData: DashboardSummary;
-  hasRunScan: boolean;
   isEligibleForFreeScan: boolean;
   type: TabType;
+  scanStatus?: Scan["status"];
```

Comment:
If we're always passing this, why would we make it optional? (Likewise for the same prop to `<View>`.)

Reply:
Right, it’s not optional anymore.